tests/projects/test_docker_projects.py

```diff
@@ -147,8 +153,13 @@ def test_docker_mount_local_artifact_uri(artifact_uri, host_artifact_uri,
     image.tags = ["image:tag"]
 
     docker_command = _get_docker_command(image, active_run)
-
-    docker_volume_expected = "-v {}:{}".format(host_artifact_uri, container_artifact_uri)
+    if os.name == 'nt' and (host_artifact_uri is not None) and \
+            (not is_valid_windows_path(host_artifact_uri)):
```

Comment:
Why do we need to check ``(host_artifact_uri is not None)` and `(not is_valid_windows_path(host_artifact_uri)`? It might be better at some point to refactor this into separate tests vs have this sort of branching logic in the test body, as it makes it hard to understand the test